docs/Advanced/SQL Searches/SQL Page Properties.md

```diff
@@ -0,0 +1,29 @@
+---
+publish: true
+---
+
+# SQL Page properties
+
+The following functions can be used in your query to get details about the page the query is running on.
+
+- notePathWithFileExtension()
+- notePath()
+- noteFileName()
+
+This will result in the following values if a query was run on this page.
+
+```json
+{  
+  "notePathWithFileExtension": "Advanced/SQL Searches/SQL Page Properties.md",  
+  "notePath": "Advanced/SQL Searches",  
+  "noteFileName": "SQL Page Properties"  
+}
+```
+
+For example if you make a page per day with the name using the pattern 'YYYY-MM-DD' you can run the following query to see all the due tasks for the day.
+
+````text
+```tasks-sql
+WHERE dueDate->isSame(noteFileName())
+```
```

Comment:
Just ran the query locally and it worked. 

![image](https://user-images.githubusercontent.com/1399443/235331335-76e7e8f7-d2e0-4e32-8787-d39190f8109e.png)

Reply:
![image](https://user-images.githubusercontent.com/1399443/235331341-a5b1b58a-7e7f-4dbf-bce0-bc842ff7d404.png)

Reply:
Updated to use .isSame(noteFileName(), 'day') to force it to day, month, year.

Reply:
if it was part of the string it would not match, as 'Notes from meeting on 2023-04-23 about the thing' != '2023-04-23'. You would need to make it a like comparison in that case or parse the file name somehow in SQL to get the date.